grid/social-media/share-post/maps/instagram.suma

```diff
@@ -0,0 +1,72 @@
+// Instagram Graph API Reference: https://developers.facebook.com/docs/instagram-api
+
+profile = "social-media/share-post@1.0"
+provider = "instagram"
+
+map SharePost {
+  apiVersion = 'v12.0'
```

Comment:
Shouldn't this be integration parameter?

Reply:
I do not think version of API should be integration parameter. When new version is released we should release new version of map, generate new test snapshots and check that tests are passing.